js/main.js

```diff
@@ -209,6 +223,65 @@ function editEntriesHandler(event) {
   const $h2FormTitle = document.querySelector('h2.form-title');
   $h2FormTitle.innerText = 'Edit Entry';
 
-}
+  // Issue#4: show the delete button
+  const $deleteButton = document.querySelector('button.delete');
+  $deleteButton.classList.remove('hidden');
 
+}
 $ulEntryList.addEventListener('click', editEntriesHandler);
+
+// Issue#4 the delete button functionality
+const $deleteButton = document.querySelector('button.delete');
+
+$deleteButton.addEventListener('click', deleteButtonHandler);
+
+function deleteButtonHandler(event) {
+  event.preventDefault();
+  toggleModal('on');
+  const $cancelButton = document.querySelector('button.button-cancel');
+  $cancelButton.addEventListener('click', () => {
+    event.preventDefault();
+    toggleModal('off');
+  });
+  const $confirmButton = document.querySelector('button.button-confirm');
+  $confirmButton.addEventListener('click', () => {
+    const currentId = data.editing.entryId;
+    deleteEntry(currentId);
+  });
```

Comment:
Move these to the global space. We don't really want to add event listeners inside of a function, because by doing that, you'll be adding an event over and over to an element every time this gets called. So you can be attaching 20 click event listeners to an element, rather than just doing one. If we do this globally, we only attach one event listener.